src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+        (e = this.state),
+        pt(t).removeEventListener("resize", e.updateBound),
+        e.scrollParents.forEach(function (t) {
+          t.removeEventListener("scroll", e.updateBound);
+        }),
+        (e.updateBound = null),
+        (e.scrollParents = []),
+        (e.scrollElement = null),
+        (e.eventsEnabled = !1),
+        e)));
+  }
+  function _t(t) {
+    return "" !== t && !isNaN(parseFloat(t)) && isFinite(t);
+  }
+  function bt(t, e) {
+    Object.keys(e).forEach(function (n) {
+      var i = "";
+      -1 !== ["width", "height", "top", "right", "bottom", "left"].indexOf(n) &&
+        _t(e[n]) &&
+        (i = "px"),
+        (t.style[n] = e[n] + i);
+    });
+  }
+  var yt = D && /Firefox/i.test(navigator.userAgent);
+  function wt(t, e, n) {
+    var i = lt(t, function (t) {
+        return t.name === e;
+      }),
+      o =
+        !!i &&
+        t.some(function (t) {
+          return t.name === n && t.enabled && t.order < i.order;
+        });
+    if (!o) {
+      var r = "`" + e + "`",
+        s = "`" + n + "`";
+      console.warn(
+        s +
+          " modifier is required by " +
+          r +
+          " modifier in order to work, be sure to include it before " +
+          r +
+          "!"
+      );
+    }
+    return o;
+  }
+  var Et = [
+      "auto-start",
+      "auto",
+      "auto-end",
+      "top-start",
+      "top",
+      "top-end",
+      "right-start",
+      "right",
+      "right-end",
+      "bottom-end",
+      "bottom",
+      "bottom-start",
+      "left-end",
+      "left",
+      "left-start",
+    ],
+    Tt = Et.slice(3);
+  function Ct(t) {
+    var e = arguments.length > 1 && void 0 !== arguments[1] && arguments[1],
+      n = Tt.indexOf(t),
+      i = Tt.slice(n + 1).concat(Tt.slice(0, n));
+    return e ? i.reverse() : i;
+  }
+  var St = "flip",
+    Dt = "clockwise",
+    kt = "counterclockwise";
+  function Nt(t, e, n, i) {
+    var o = [0, 0],
+      r = -1 !== ["right", "left"].indexOf(i),
+      s = t.split(/(\+|\-)/).map(function (t) {
+        return t.trim();
+      }),
+      a = s.indexOf(
+        lt(s, function (t) {
+          return -1 !== t.search(/,|\s/);
+        })
+      );
+    s[a] &&
+      -1 === s[a].indexOf(",") &&
+      console.warn(
+        "Offsets separated by white space(s) are deprecated, use a comma (,) instead."
+      );
+    var l = /\s*,\s*|\s+/,
+      c =
+        -1 !== a
+          ? [
+              s.slice(0, a).concat([s[a].split(l)[0]]),
+              [s[a].split(l)[1]].concat(s.slice(a + 1)),
+            ]
+          : [s];
+    return (
+      (c = c.map(function (t, i) {
+        var o = (1 === i ? !r : r) ? "height" : "width",
+          s = !1;
+        return t
+          .reduce(function (t, e) {
+            return "" === t[t.length - 1] && -1 !== ["+", "-"].indexOf(e)
+              ? ((t[t.length - 1] = e), (s = !0), t)
+              : s
+              ? ((t[t.length - 1] += e), (s = !1), t)
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)